app/models/tagging.rb

```diff
@@ -46,9 +46,11 @@ def update_taggings_count
   end
 
   def update_search
-    reindex_boundary = 100
-    if %w(Character Relationship Freeform).include?(tagger.type)
-      tagger.enqueue_to_index if tagger.taggings_count < reindex_boundary
-    end
+    # When a tag's count is more than this threshold, we start caching it,
+    # so taggings_count becomes less accurate. That's our cue to stop
+    # eagerly reindexing counts as well.
+    return unless %w[Fandom Character Relationship Freeform].include?(tagger.type)
```

Comment:
Could use Tag::USER_DEFINED here, now that the check isn't omitting Fandom anymore.